build.assets/Dockerfile-multiarch

```diff
@@ -44,6 +57,7 @@ ARG GOLANG_VERSION
 ARG RUST_VERSION
 ARG UID
 ARG GID
+ARG RUSTUP_VERSION=1.25.2
```

Comment:
Do we need to pin the rustup version? Thus far we've pinned the Rust version, but used the latest version of rustup at time of install.

Reply:
Sure. Sorry, that's a leftover from an experiment, removed.